src/main/java/com/github/hcsp/controlflow/Main.java

```diff
@@ -12,7 +12,7 @@
     public static void printOddNumbersBetween(int start, int end) {
         for (int i = start; i <= end; i++) {
             if (i % 2 != 0) {
-                System.out.println(i + ",");
+                System.out.print(i + ",");
```

Comment:
WTF! But now I know the difference between `system.out.println` and `system.out.print()`